### DISCUSSION TITLE
Create Quality control plots

### DISCUSSION THREAD

**Discussion by jindalk** [GreenleafLab/ArchR#1487] [Questions / Documentation] [answered]
Is there a way to create quality control plots like these from an ArchR object? I tried the ggPoint function but wasn't sure how to overlay density colors. https://www.archrproject.com/bookdown/per-cell-quality-control.html

**Comment by rcorces** [comment, marked as answer]
This is the code used to make the plot you're talking about:
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/CreateArrow.R#L519-L532
